test/common/router/scoped_rds_test.cc

```diff
@@ -167,6 +224,10 @@ route_configuration_name: foo_routes
   parseScopedRouteConfigurationFromYaml(*resources.Add(), config_yaml);
   EXPECT_THROW_WITH_MESSAGE(subscription_callbacks_->onConfigUpdate(resources, "1"), EnvoyException,
                             "duplicate scoped route configuration foo_scope found");
+
+  EXPECT_THROW_WITH_MESSAGE(
```

Comment:
Let's match CDS behavior and validate it; i.e., the first instance of every duplicate resource should be applied.